### PR TITLE
Fix a few warnings

### PR DESCRIPTION
**Pull request by metalgearsloth**

### CODE REVIEW

**Comment by 0x6273** [comment]
Why did you change it to Postgres? Is the SQLite db engine being deprecated?

File: Content.Shared/CCVar/CCVars.cs [hunk 1]
```diff
@@ -359,7 +359,7 @@ public sealed class CCVars : CVars
          */
 
         public static readonly CVarDef<string> DatabaseEngine =
-            CVarDef.Create("database.engine", "sqlite", CVar.SERVERONLY);
+            CVarDef.Create("database.engine", "postgres", CVar.SERVERONLY);
```
Comment (Zumorica) [inline Content.Shared/CCVar/CCVars.cs:362]: I think you commited this by mistake

File: Content.Shared/CCVar/CCVars.cs [hunk 2]
```diff
@@ -392,10 +392,10 @@ public sealed class CCVars : CVars
             CVarDef.Create("database.pg_database", "ss14", CVar.SERVERONLY);
 
         public static readonly CVarDef<string> DatabasePgUsername =
-            CVarDef.Create("database.pg_username", "", CVar.SERVERONLY);
+            CVarDef.Create("database.pg_username", "postgres", CVar.SERVERONLY);
 
         public static readonly CVarDef<string> DatabasePgPassword =
-            CVarDef.Create("database.pg_password", "", CVar.SERVERONLY | CVar.CONFIDENTIAL);
+            CVarDef.Create("database.pg_password", "Ohiyome2", CVar.SERVERONLY | CVar.CONFIDENTIAL);
```
Comment (Zumorica) [inline Content.Shared/CCVar/CCVars.cs:398]: ditto

**Review by Zumorica — CHANGES_REQUESTED** [review]

**Comment by metalgearsloth** [comment]
Cleared my cvars and forgot to clear my diff.

**Pull request merged**